run-checks

```diff
@@ -218,7 +218,7 @@ if [ "$STATIC" = 1 ]; then
     echo Checking all interfaces have minimal spread test
     missing_interface_spread_test
 
-    if command -v staticcheck >/dev/null || [[ "$(go version)" =~ go1\.(9|1[0-9])\.[0-9] ]]; then
+    if command -v staticcheck >/dev/null || ( go version | grep -E 'go1\.(9|1[0-9])\.[0-9]' ); then
```

Comment:
You probably meant `grep -E 'go1\.(9|1[0-9])(\.[0-9])?'` since Go 1.10 reported `go version go1.10 linux..`

Reply:
Looks good.